acapy_agent/wallet/crypto.py

```diff
@@ -63,20 +65,24 @@ def create_ed25519_keypair(seed: Optional[bytes] = None) -> Tuple[bytes, bytes]:
     return pk, sk
 
 
-def seed_to_did(seed: str) -> str:
+def seed_to_did(seed: str, method: Optional[DIDMethod] = SOV) -> str:
     """Derive a DID from a seed value.
 
     Args:
         seed: The seed to derive
+        method: The DID method to use
 
     Returns:
         The DID derived from the seed
 
     """
     seed = validate_seed(seed)
     verkey, _ = create_ed25519_keypair(seed)
-    did = bytes_to_b58(verkey[:16])
-    return did
+    if method == SOV:
+        return bytes_to_b58(verkey[:16])
+    if method == INDY:
+        return f"did:indy:{bytes_to_b58(hashlib.sha256(verkey).hexdigest()[:16])}"
```

Comment:
why hexdigest rather than digest ?

Reply:
I wasn't aware askar used digest, and honestly not super confident about some stuff I'm doing here. I'll change it to digest and make sure it's working.